### PR TITLE
Data loading fix for [Part 1 - Basic Features]

### PR DESCRIPTION
**Pull request by dphov**
Data loading fix for [Part 1 - Basic Features],
Updating dependencies

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@dphov Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-guides/tut-react-and-spring-data-rest&pullRequestId=60)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-guides/tut-react-and-spring-data-rest&pullRequestId=60) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@dphov Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-guides/tut-react-and-spring-data-rest&pullRequestId=60)!

**Comment by gregturn** [comment]
Thanks for the PR

*With when.js, `then` must return another promise. You use `done` when there is nothing else to be done. 
* Your change only updates the "basic" version.  It would be nice to update them all. 
* it's hard to read all the mods because you also changed all the formatting from tabs to spaces. I'd rather keep it the way it is. We can discuss that in a separate issue.

File: basic/src/main/js/app.js [hunk 1]
```diff
@@ -1,73 +1,72 @@
...
-						<th>First Name</th>
-						<th>Last Name</th>
-						<th>Description</th>
-					</tr>
-					{employees}
-				</tbody>
-			</table>
-		)
-	}
+class EmployeeList extends React.Component {
```
Comment (Silwest) [inline basic/src/main/js/app.js]: Just to be consistent shouldn't it be `extend Component`.

**Comment by gregturn** [comment]
This looks very nice except

* you changed the spacing from tabs to double spaces. I'd like to not make such a visual change. 
* it only updates the basic module.

**Pull request closed without merging**